lib/stdext.mli

```diff
@@ -25,10 +25,10 @@ module Path : sig
       - ends with ['~']
       - or begins with ['#'] and ends with ['#']. *)
 
-  val find_files : name_wo_ext:string -> Fpath.t list -> Fpath.t list
+  val find_files : name_wo_ext:string list -> Fpath.t list -> Fpath.t list
```

Comment:
Might be worth renaming the argument to `names_wo_ext`.

Reply:
Agreed.